tests/rptest/services/redpanda.py

```diff
@@ -574,14 +576,34 @@ def start_node(self,
         if self.coproc_enabled():
             self.start_wasm_engine(node)
 
+        def is_status_ready():
+            status = None
+            try:
+                status = Admin.ready(node).get("status")
+            except:
+                self.logger.error(
```

Comment:
As in previous commit, log.exception is generally preferable to calling into `traceback`.